api/RTCRtpContributingSource.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "RTCRtpContributingSource": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpContributingSource",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "59"
```

Comment:
In the wiki there is "[1] Disabled by default in Firefox 59 due to changes that occurred in the specification after being implemented. To enable the non-compliant implementation, set the preference media.peerconnection.rtpsourcesapi.enable to true."

Why is this information not added here? (as a `flags` object)
The same for `RTCRtpSynchronizationSource`.